newsfragments/8626.feature.rst

```diff
@@ -0,0 +1,5 @@
+Add server option ``--proxy-trusted-addresses`` to enabled parsing of proxy headers from trusted addresses.
+By default, the server will trust the proxy headers from localhost
+
+.. tip::
+  You can also configure the trusted addresses using the environment variable ``PARSEC_PROXY_TRUSTED_ADDRESSES``.
```

Comment:
I reiterate my previous outdated comment:

We should still stick to one-liner without fancy formatting, as it is the case for [almost all changelog entries](https://github.com/Scille/parsec-cloud/blob/master/HISTORY.rst?rgh-link-date=2024-10-09T10%3A09%3A28Z). If more detail is needed, it should be added to the issue referenced by the news fragment (https://github.com/Scille/parsec-cloud/issues/8626) which will be displayed in the changelog.

```suggestion
Add server option ``--proxy-trusted-addresses`` to enable parsing of proxy headers from trusted addresses. By default, the server will trust the proxy headers from localhost
```